tools/topology/topology1/sof-tgl-rt711-rt1308.m4

```diff
@@ -178,11 +165,11 @@ dnl     buffer, periods, format,
 dnl     deadline, priority, core, time_domain)
 
 # playback DAI is ALH(SDW0 PIN2) using 2 periods
-# Buffers use s24le format, with 48 frame per 1000us on core 0 with priority 0
-DAI_ADD(sof/pipe-dai-playback.m4,
+# Buffers use s32le format, with 48 frame per 1000us on core 0 with priority 0
+DAI_ADD(sof/pipe-mixer-dai-playback.m4,
 	1, ALH, 2, SDW0-Playback,
-	PIPELINE_SOURCE_1, 2, s24le,
-	1000, 0, 0, SCHEDULE_TIME_DOMAIN_TIMER)
+	NOT_USED_IGNORED, 2, s24le,
```

Comment:
I am confused by s32le in comments and s24le here, is this unrelated ? thanks

Reply:
yes, there is some copy/paste mistake here, will fix. thanks for noticing this @RanderWang